README.rst

```diff
@@ -59,24 +64,24 @@ Supervised Manual Drupal Update
 -------------------------------
 
 It is possible to check for and install updates from the Drupal Admin
-UI:: **Admin > Reports > Avaialble Updates**
+UI:: **Admin > Reports > Available Updates**
 
 Updates for Drupal9 Core often need to be done via commandline. For full
 details, please consult the `Drupal8 Upgrade docs`_, we recommend using
 `Composer to update from the commandline`_::
 
     cd /var/www/drupal9
-    # update composer - not strictly neccessary
+    # update composer - not strictly necessary
     composer self-update
```

Comment:
Does this work without the turnkey wrapper script?

Reply:
You need to use composer directly (as `root`) to use the `self-update` command. Otherwise you won't have permission.

Is that what you mean?

Reply:
Yep in that case I'm happy with this